src/tests/orders.spec.js

```diff
@@ -0,0 +1,124 @@
+import { expect, test } from "@playwright/test";
+import { ApplicationPages } from "./pages/application.pages.js";
+import { OrdersPage } from "./pages/orders.page.js";
+import { icoNumber, clientName, clientAddress,
+    substitute, contactName, contactPhone, contactEmail,
+    startDate1, endDate1, startDate2, endDate2, startDate3, endDate3,
+    campStudentsNumber, campStudentsAge, campAdultsNumber,
+    natureStudentsNumber, natureStudentsAge, natureAdultsNumber, natureStartTime, natureEndTime,
+    orderConfirmationTitle
+ } from "../fixtures/fixtures.js";
+
+test.describe("Orders", async () => {
+    let application;
+    let order;
+
+    test.describe("Navigation to Orders Page", async () => {
+
+        test.beforeEach(async ({ page }) => {
+            application = new ApplicationPages(page);
+            await application.open();
+        });
+
+        test("User can navigate to Orders for schools from 'For Teachers' menu item", async ({ page }) => {
```

Comment:
Tady podle mě popis testu neodpovídá. Testuje se jen, že je vidět _newOrderLocator_ (menu item položka).
Jelikož _page_ není použita tak ji netřeba uvádět (tp samé v některých dalěích testech).
```suggestion
        test("User can navigate to Orders for schools from 'For Teachers' menu item", async () => {
```